### ISSUE TITLE
Matomo 4.6.0 update causes integrity check error

### ISSUE THREAD

**Issue by MichaIng** [piwik/piwik#18412]
```
Files were found in your Matomo, but we didn't expect them.
--> Please delete these files to prevent errors. <--

File to delete: vendor/lox/xhprof/bin/xhprofile
```
This already happened at least on last RC version.

## Expected Behavior
If there were no integrity check errors before an update, there shouldn't be any after an update.

## Current Behavior
Update to Matomo 4.6.0 creates `vendor/lox/xhprof/bin/xhprofile` which violates the integrity check.

## Possible Solution
Remove this file from the update archive.

## Steps to Reproduce (for Bugs)

1. Start with any Matomo <4.6.0
2. Verify that there are no integrity check errors, else fix them
3. Update to Matomo 4.6.0
4. Check for integrity check errors

## Your Environment

* Matomo Version: 4.6.0
* PHP Version: 8.1.0
* Server Operating System: Debian Bookworm
* Additionally installed plugins:
```
API, Actions, Annotations, BulkTracking, CoreAdminHome, CoreConsole, CoreHome, CorePluginsAdmin, CoreUpdater, CoreVisualizations, CoreVue, DBStats, DarkTheme 1.1.7, Dashboard, DevicePlugins, DevicesDetection, Diagnostics, Goals, ImageGraph, Insights, Installation, Intl, LanguagesManager, Live, LogViewer 4.0.1, Login, Marketplace, Monolog, Morpheus, PagePerformance, PrivacyManager, Proxy, Referrers, Resolution, SEO, SegmentEditor, SitesManager, TrackingSpamPrevention 4.1.0, Transitions, UserLanguage, UsersManager, VisitFrequency, VisitTime, VisitorInterest, VisitsSummary, WebsiteMeasurable
```

**Comment by DBX12** [comment]
I'm seeing this as well. Additionally, I got some more files highlighted by the integrity check

```
Files were found in your Matomo, but we didn't expect them.
--> Please delete these files to prevent errors. <--

File to delete: misc/composer/build-xhprof.sh
File to delete: misc/composer/clean-xhprof.sh
File to delete: vendor/lox/xhprof/bin/xhprofile
File to delete: node_modules/jquery.dotdotdot/.npmignore
```
While the first two seem like build dependencies and could potentially be deleted, the third one looks definitely like a wanted file to me. That `.npmignore` is _probably_ unproblematic to delete.

**Comment by MichaIng** [comment]
At least the latest update does not ship the other three files, but probably they were shipped with Matomo 4.5 already? I remember seeing integrity check errors more regularly after updates, but I didn't report them every time when faced during beta or RC updates. Now it is however a stable release and that additional file was definitely shipped with the v4.6.0 stable update since I removed the same file before on v4.6.0-rc2.

**Comment by tsteur** [comment]
Thanks for this @MichaIng . I'll mark this as a duplicate of https://github.com/matomo-org/matomo/issues/18415 we will look into this soon and I think we'll release a 4.6.1 where this will be fixed.
